tests/test_evaluation.py

```
@@ -8,6 +8,23 @@
     compute_oks,
 )
 from sleap_nn.evaluation import Evaluator
+from loguru import logger
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)